src/test/java/edu/cornell/kfs/concur/batch/fixture/PdpFeedFileBaseEntryFixture.java

```diff
@@ -12,92 +15,42 @@
 import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedHeaderEntry;
 import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedPayeeIdEntry;
 import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedTrailerEntry;
-
-public class PdpFeedFileBaseEntryFixture {
-    public static PdpFeedFileBaseEntry buildPdpFile() {
-        PdpFeedFileBaseEntry pdpFile = new PdpFeedFileBaseEntry();
-        pdpFile.setHeader(buildHeader());
-        pdpFile.setVersion("1.0");
-        pdpFile.setTrailer(buildTrailer());
-        pdpFile.setGroup(buildGroups());
-        return pdpFile;
-    }
-
-    private static PdpFeedHeaderEntry buildHeader() {
-        PdpFeedHeaderEntry header = new PdpFeedHeaderEntry();
-        header.setChart("IT");
-        header.setCreationDate("02/17/2017");
-        header.setSubUnit("CLIF");
-        header.setUnit("CRNL");
-        return header;
-    }
-
-    private static PdpFeedTrailerEntry buildTrailer() {
-        PdpFeedTrailerEntry trailer = new PdpFeedTrailerEntry();
-        trailer.setDetailCount(new Integer(1));
-        trailer.setDetailTotAmt(new KualiDecimal(96.95));
-        return trailer;
-    }
-
-    private static List<PdpFeedGroupEntry> buildGroups() {
-        List<PdpFeedGroupEntry> groups = new ArrayList();
-        groups.add(buildGroup("Bimbo Foods Inc"));
-        return groups;
+import edu.cornell.kfs.concur.ConcurTestConstants.PdpFeedFileConstants;
+
+public enum PdpFeedFileBaseEntryFixture {
+
+    MARSHAL_TEST(EnumSet.of(PdpFeedGroupEntryFixture.MARSHAL_TEST)),
+    FEED_ONE_TRANS_ZERO(EnumSet.of(PdpFeedGroupEntryFixture.GROUP_ONE_TRANS_ZERO)),
+    FEED_ONE_TRANS_ZER_ONE_TRANS_POSITIVE(EnumSet.of(PdpFeedGroupEntryFixture.GROUP_ONE_TRANS_ZERO_ONE_TRANS_POSITIVE)),
+    FEED_TWO_TRANS_SUM_TO_POSITIVE(EnumSet.of(PdpFeedGroupEntryFixture.GROUP_TWO_TRANS_SUM_TO_POSITIVE)),
+    FEED_TWO_TRANS_SUM_TO_ZERO(EnumSet.of(PdpFeedGroupEntryFixture.GROUP_TWO_TRANS_SUM_TO_ZERO)),
+    FEED_TWO_TRANS_SUM_TO_NEGATIVE(EnumSet.of(PdpFeedGroupEntryFixture.GROUP_TWO_TRANS_SUM_TO_NEGATIVE)),
+    FEED_TWO_DETAILS_ONE_POSITIVE_ONE_ZERO(EnumSet.of(PdpFeedGroupEntryFixture.GROUP_TWO_DETAILS_ONE_POSITIVE_ONE_ZERO)),
+    FEED_TWO_GROUPS_ONE_ZERO(EnumSet.of(PdpFeedGroupEntryFixture.GROUP_TWO_TRANS_SUM_TO_ZERO, PdpFeedGroupEntryFixture.GROUP_TWO_TRANS_SUM_TO_POSITIVE));
+    
+    public final PdpFeedHeaderEntryFixture headerEntry;
+    public final EnumSet<PdpFeedGroupEntryFixture> groupEntries;
```

Comment:
Another area where it would be helpful to wrap the EnumSet inside of an immutable Set.